roles/nova-common/templates/etc/serverspec/nova_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'spec_helper'
+
+files = { "api-paste.ini"=> 640, "nova.conf"=> 640, "policy.json"=> 644 ,"rootwrap.conf"=> 640 }
+files.each do |file, mode|
+  describe file("/etc/nova/#{file}") do
+    it { should be_mode mode }
+    it { should be_owned_by 'nova' }
+    it { should be_grouped_into 'nova' }
+  end
+end
+
+files = ['api-metadata.filters' , 'baremetal-compute-ipmi.filters' , 'baremetal-deploy-helper.filters' , 'compute.filters', 'network.filters']
+files.each do |file|
+  describe file("/etc/nova/rootwrap.d/#{file}") do
+    it { should be_owned_by 'root' }
+    it { should be_grouped_into 'root' }
+    it { should be_writable.by('owner') }
+  end
+end   
+
+describe file('/etc/nova/rootwrap.d') do
+    it { should be_owned_by 'root' }
+    it { should be_grouped_into 'root' }
+end
+
+files = {'nova-api.log'=> 'nova', 'nova-cert.log'=> 'nova' , 'nova-conductor.log'=> 'nova', 'nova-consoleauth.log'=> 'nova', 'nova-manage.log'=> 'root', 'nova-novncproxy.log'=> 'nova' , 'nova-scheduler.log'=> 'nova'}
```

Comment:
These files aren't going to exist on every nova host. Since you have these being applied through nova-common, on a compute only node many of these files won't exist.

I also don't see nova-compute.log in this list.